### PR TITLE
Feature/default transactional operations disabled

### PR DESCRIPTION
**Pull request by CarstenHollmann**
The transactional operations are deactivated by default

### CODE REVIEW

File: core/api/src/main/java/org/n52/sos/request/operator/RequestOperatorKey.java [hunk 1]
```diff
@@ -42,15 +42,27 @@
     private final ServiceOperatorKey sok;
 
     private final String operationName;
+    
+    private boolean defaultActive = true;
```
Comment (autermann) [inline core/api/src/main/java/org/n52/sos/request/operator/RequestOperatorKey.java]: Can't this be `final`?

**Comment by CarstenHollmann** [comment]
Updated the RequestOperatorKey.java, defaultActive is now final.

**Comment by meliezer** [comment]
I see at admin/settings#transactional_security that only the non-localhost transactional operations are deactivated.

**Comment by EHJ-52n** [comment]
@meliezer 

The default configuration is now that __all transactional operations__ are __deactivated__ by default. In addition, the __transactional security layer__ is __activated__ by default and __limited__ to __127.0.0.1__ (aka localhost). Hence, by activating any transactional operation you still have the limitation to localhost.

Hope this clarifies the situation a bit for you. Do you have any recommendations regarding the help texts in the GUI to make this more clear for new users?

**Comment by meliezer** [comment]
It's not clear. Are you saying that I should see InsertSensor (for example) as disabled under admin/operations right after installation? It's not.
What exactly should do a new user? Activate the desired transactional operations (how?) and then if required change the configuration of the transactional security layer?

**Comment by EHJ-52n** [comment]
> It's not clear. Are you saying that I should see InsertSensor (for example) as disabled under  admin/operations right after installation? It's not.

Yes it should. Which version do you use? The latest development, or the latest stable?


> What exactly should do a new user? Activate the desired transactional operations (how?) and then if required change the configuration of the transactional security layer?

Yes. There is a information on the test client page providing direct links to the places where to adjust the transactional security layer and enable the required operations.

**Comment by meliezer** [comment]
>Yes it should. Which version do you use? The latest development, or the latest stable?
Latest stable, but I may be wrong. It's not clear if my colleague had changed it.
>Yes. There is a information on the test client page providing direct links to the places where to >adjust the transactional security layer and enable the required operations. 
You are right. I haven't seen it. It's very clear.
You can remove my comments above. Sorry.

**Comment by EHJ-52n** [comment]
@meliezer Thanks for feedback. I just updated your last comment to get a better layout and make it easier to understand.

**Pull request merged**